website/blog/2023-08-14-ferretdb-new-release-v180.md

```diff
@@ -0,0 +1,57 @@
+---
+slug: ferretdb-new-release-v180
+title: FerretDB v1.8.0 – New release
+authors: [alex]
+description: >
+  FerretDB v1.8.0 has just been released with new features, bug fixes, and improvements to the SQLite and PostgreSQL backend.
+image: /img/blog/ferretdb-v1.8.0.jpg
+tags: [release]
+---
+
+![FerretDB v.1.8.0 - new release](/img/blog/ferretdb-v1.8.0.jpg)
+
+FerretDB v1.8.0 has just been released with new features, bug fixes, and improvements to the SQLite and PostgreSQL backends.
+
+<!--truncate-->
+
+In the last few weeks we've added support for the `$group` stage `_id` expression and the `$expr` evaluation query operator, and also made several optimizations and resolved issues with some of the previously implemented commands, our integration tests, among others.
+
+These are quiet times for many people across the globe, including FerretDB maintainers and contributors, as we enjoy the summer vacations and the warm weather.
+
+At [FerretDB](https://www.ferretdb.io), we see this period as an exciting one!
+For instance, with the latest improvements and additions in FerretDB v1.8.0, we are getting closer to the production-ready release of the SQLite backend for FerretDB, which should be available soon.
+In addition to that, we'll be attending [Postgres Ibiza 2023 (August 29-31, 2023)](https://pgibz.io/) and [Civo Navigate (September 5-6, 2023)](https://www.civo.com/navigate).
+Our CEO, Peter Farkas, will be giving a presentation at both events – we hope to see you there!
+
+Without further ado, let's check out some of the notable features in this release!
+
+## New features
+
+In the new release, we've added support for the `$group` stage `_id` expression.
+
+With this update, users can now use more complex document structures for the `_id` field in `$group` aggregations; you can now group data using an expression like `{"$group": {_id: {"v": "$v"}}}`.
+
+We've also added support for the `$expr` evaluation query operator, which allows you to use expressions within the `$match` aggregation stage, and the `find` command.
+For example, you can now execute queries like `db.values.find( { $expr: <expression> } )`.
+For the `$match` aggregation stage, you can use it like this: `db.col.aggregate({<aggregation expression>})`.
```

Comment:
There is no `$match` stage in the example that supposedly demonstrates the usage of it.

Please do your own proofreading and review first.